Devscord.DiscordFramework/Framework/Commands/Services/BotCommandsMatchingService.cs

```diff
@@ -79,7 +79,7 @@ private bool IsMatchedPropertyType(string value, BotCommandPropertyType type)
             {
                 return false;
             }
-            if (type == BotCommandPropertyType.SingleWord && value.Contains(' '))
+            if (type == BotCommandPropertyType.SingleWord && (value?.Contains(' ') ?? false))
```

Comment:
?

Reply:
dzięki temu mogę podać argument -all bez 
![1](https://user-images.githubusercontent.com/47067171/87725187-15666580-c7bd-11ea-92e9-40241e73f07c.jpg)
zawartości

Reply:
zrobiłem w ten sposób, że jak ICommand ma jakiś optional argument i się go nie poda do jest null, jak się poda z argumentem to jest argument, a jak się poda bez argumentu to jest pusty string

Reply:
no bo nie oznaczyłeś argumentu all jako bool
jeśli dodasz atrybut [Bool] to będzie działać (wtedy [Optional] nie jest już potrzebny)

Reply:
dalej nie działa
![1](https://user-images.githubusercontent.com/47067171/87725864-59a63580-c7be-11ea-852f-a603cd145e63.jpg)

Reply:
bo ma być jeszcze atrybut [Bool]
zobacz jak to jest w przypadku HelpCommand

Reply:
o dzięki działa